UnitTests/GitUI.Tests/UserControls/BlameControlTests.cs

```diff
@@ -149,5 +150,105 @@ public void BuildBlameContents_WithDateButNotTime()
                 AppSettings.BlameShowAuthorTime = originalValue;
             }
         }
+
+        private IEnumerable<GitBlameLine> CreateBlameLine(params DateTime[] lineDates)
+        {
+            for (var index = 0; index < lineDates.Length; index++)
+            {
+                DateTime lineDate = lineDates[index];
+                yield return new GitBlameLine(
+                    new GitBlameCommit(null, "Author1", "@Author1", lineDate, string.Empty,
+                        "Commiter", "@Committer", lineDate, string.Empty, "Summary1", "file"),
+                    index + 1, index + 1, "text");
+            }
+        }
+
+        [Test]
+        public void CalculateBlameGutterData_When_date_is_older_than_artificial_old_boundary_Then_it_define_first_age_bucket()
+        {
+            // Given
+            IReadOnlyList<GitBlameLine> blameLines = CreateBlameLine(new DateTime(2008, 9, 1)).ToList();
```

Comment:
`ArtificialOldBoundary` is relative to current clock, while `new DateTime(2008, 9, 1)` is absolute.

one way to resolve it would be: 
```csharp
var olderDate = _blameControl.GetTestAccessor().ArtificialOldBoundary.AddYears(-1)
```